build.rs

```diff
@@ -45,16 +42,15 @@ fn main() {
         println!("cargo:rustc-link-lib=c++");
     }
 
-    println!("cargo:rerun-if-changed=wrapper.h");
-
     let mut bindings = bindgen::Builder::default()
         .clang_arg("-xc++")
         .header("wrapper.h")
-        .parse_callbacks(Box::new(bindgen::CargoCallbacks))
+        .parse_callbacks(Box::new(bindgen::CargoCallbacks::new()))
         .derive_partialeq(true)
         .derive_eq(true)
         .derive_hash(true)
-        .derive_debug(true);
+        .derive_debug(true)
+        .blocklist_item("std::value");
```

Comment:
Without this last line I'm getting errors with the bindings on my machine:

> error[E0412]: cannot find type `_Tp` in this scope
  --> /Users/tomaka/Projets/astcenc-sys/target/debug/build/astcenc-sys-6f5ac5488e172a76/out/bindings.rs:93:27
   |
93 |     pub static std_value: _Tp;
   |                           ^^^ not found in this scope

I've found the fix here: https://github.com/rust-lang/rust-bindgen/issues/1738#issuecomment-2052187692